components/prism-nginx.js

```diff
@@ -0,0 +1,18 @@
+});
+
+Prism.languages.insertBefore('nginx', 'keyword', {
+        variable:/(\$[a-zA-Z_]+)/ig,
+        'package': {
+                pattern: /(\\|namespace\s+|use\s+)[\w\\]+/,
+                lookbehind: !0,
```

Comment:
Please use `true` here. We can leave this kind of compression for the generated minified version.